__tests__/analysis.test.js

```diff
@@ -0,0 +1,27 @@
+import analyze from '../src/analysis';
+
+test('should return an object', () => {
+  expect(typeof analyze([1, 8, 3, 4, 2, 6])).toBe('object');
```

Comment:
- [ ] [OPTIONAL] You are only analyzing this one array for all methods, test the code with another array too, and even with an empty array to make sure your code is not breaking.

- [ ] [OPTIONAL] Instead of repeating the same array over and over again you can save it in a const and reuse it.